### ISSUE TITLE
How to write posts in markdown?

### ISSUE THREAD

**Issue by vrthra** [toddsundsted/ktistec#82]
The change log [here] says it now supports "Render to HTML posts when content is Markdown". How do I take advantage of this? I tried [posting](https://gopinath.org/objects/kLdLDnmFVAE) with markdown, but the content seems to remain in text.

**Comment by toddsundsted** [comment]
@vrthra there's support for rendering markdown content to html, but currently no support (no user-interface) for authoring in markdown. it's not conceptually difficult to imaging adding, but it's not trivial, either.

**Comment by vrthra** [comment]
Thanks @toddsundsted for the clarification. To understand this a bit further, What would such a user-interface look like? Would it be something like the interface here in Github comments? (a plain text box, and a few buttons on the toolbar that modifies the text directly, and a preview)?

**Comment by toddsundsted** [comment]
yes, but even that has interesting edge cases—uploading images, for example, and preview, as you mentioned. the super-easy case would be just a text entry box for markdown text with no affordances for generating headings, etc.

that would actually work, and be pretty easy to implement, but that's pretty far off from what people expect—which is more like what github and others provide.

**Comment by JayVii** [comment]
Writing posts in markdown would be great! It may also solve some other requests like #52 as markdown supports it natively:
```
![alt text](image url "image Title")
```

I do get that a proper markdown textfield with image upload etc may be non-trivial to implement, unless there is a framework similar to the trix-editor that is currently used by Ktistec
